tests/models/test_provider.py

```diff
@@ -236,16 +238,18 @@ def test_to_list_of_properties():
 
 
 @pytest.fixture
-def valid_namespace():
+def valid_namespace() -> Literal["namespace"]:
```

Comment:
I would just use `str` here and below. I think the point of using a fixture is to be able to change it without updating all uses.